### PR TITLE
Issue/remote agent docs

### PR DESCRIPTION
**Pull request by wouterdb**
# Description

added instructions about passwordless sudo to remote agent setup 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] ~~Attached issue to pull request~~
- [x] Changelog entry
- [ ] ~~Type annotations are present~~
- [ ] ~~Code is clear and sufficiently documented~~
- [ ] ~~No (preventable) type errors (check using make mypy or make mypy-diff)~~
- [ ] ~~Sufficient test cases (reproduces the bug/tests the requested feature)~~
- [ ] ~~Correct, in line with design~~
- [x] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: docs/install/2-configure-agents.rst [hunk 1]
```diff
@@ -22,6 +22,8 @@ with respect to the Inmanta server and the agent has to execute I/O operations o
 * The Inmanta server should have passphraseless SSH access on the machine it maps to. More information on how to set up SSH
   connectivity can be found at :ref:`configure_server_step_6`
 * The remote machine should have a Python 2 or 3 interpreter installed. The binary executed by default is ``python``.
+* The remote user should either be ``root`` or have the ability to do a passwordless sudo.
+  To enable passwordless sudo, add a file to ``/etc/sudoers.d/`` containing ``username ALL=(ALL) NOPASSWD: ALL``.
```
Comment (arnaudsjs) [inline docs/install/2-configure-agents.rst:26]:
```suggestion
  To enable passwordless sudo  for the user ``username``, add a file to ``/etc/sudoers.d/`` containing ``username ALL=(ALL) NOPASSWD: ALL``.
```

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: docs/install/2-configure-agents.rst (hunk 1 above)
Comment (arnaudsjs) [reply docs/install/2-configure-agents.rst:26]: Should we also mention that you should edit the `sudoers` file via `visudo` or do we assume the reader knows this?

Comment (wouterdb) [reply docs/install/2-configure-agents.rst:26]: I didn't know either

Comment (bartv) [reply docs/install/2-configure-agents.rst:26]: referer to the sudo documentation?

Comment (wouterdb) [reply docs/install/2-configure-agents.rst]: added

**Review by arnaudsjs — APPROVED** [review]

**Pull request closed without merging**